WooCommerce/Classes/ViewModels/OrderDetailsViewModel/OrderDetailsDataSource+Constants.swift

```diff
@@ -0,0 +1,140 @@
+import Foundation
+import Yosemite
+
+
+// MARK: - Constants
+extension OrderDetailsDataSource {
```

Comment:
could these constants also go back to the main file and back to private?

Reply:
yep! I thought we had other files where the Constants were kept separately, but searching the project shows we don't.